<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InsertPathDlg</class>
 <widget class="QDialog" name="InsertPathDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>125</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>InsertPathDlg</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditPtX">
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLineEdit" name="lineEditPtY">
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLineEdit" name="lineEditPtZ">
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>A:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEditPtA">
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>B:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="lineEditPtB">
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>C:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QLineEdit" name="lineEditPtC">
     <property name="minimumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>68</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>姿态类型:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="5">
    <widget class="QComboBox" name="comboBox">
     <property name="minimumSize">
      <size>
       <width>220</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>220</width>
       <height>22</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="minimumSize">
      <size>
       <width>160</width>
       <height>22</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>22</height>
      </size>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>lineEditPtX</tabstop>
  <tabstop>lineEditPtY</tabstop>
  <tabstop>lineEditPtZ</tabstop>
  <tabstop>lineEditPtA</tabstop>
  <tabstop>lineEditPtB</tabstop>
  <tabstop>lineEditPtC</tabstop>
  <tabstop>comboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
